fix a bunch of warnings from GCC 4.4

diff -ur grip-3.3.1/src/discdb.c grip-3.3.1_warn/src/discdb.c
--- grip-3.3.1/src/discdb.c	2005-06-25 20:23:13.000000000 +0200
+++ grip-3.3.1_warn/src/discdb.c	2010-10-17 22:08:44.000000000 +0200
@@ -45,10 +45,10 @@
 #include "config.h"
 
 extern char *Program;
-static char *StrConvertEncoding(char *str,char *from,char *to,int max_len);
+static char *StrConvertEncoding(char *str,char const*from,char *to,int max_len);
 gboolean DiscDBUTF8Validate(const DiscInfo *disc,const DiscData *data);
 static void DiscDBConvertEncoding(DiscInfo *disc,DiscData *data,
-                                  char *from,char *to);
+                                  char const*from,char *to);
 static int DiscDBSum(int val);
 static char *DiscDBReadLine(char **dataptr);
 static GString *DiscDBMakeURI(DiscDBServer *server,DiscDBHello *hello,
@@ -496,7 +496,7 @@
   }
 }
 
-static char *StrConvertEncoding(char *str,char *from,char *to,int max_len)
+static char *StrConvertEncoding(char *str,char const*from,char *to,int max_len)
 {
   char *conv_str;
   gsize rb,wb;
@@ -541,7 +541,7 @@
 
 
 static void DiscDBConvertEncoding(DiscInfo *disc,DiscData *data,
-                                  char *from,char *to)
+                                  char const*from,char *to)
 {
   int track;
 
diff -ur grip-3.3.1/src/status_window.c grip-3.3.1_warn/src/status_window.c
--- grip-3.3.1/src/status_window.c	2004-04-15 20:21:37.000000000 +0200
+++ grip-3.3.1_warn/src/status_window.c	2010-10-17 22:06:43.000000000 +0200
@@ -22,6 +22,7 @@
 
 #include <stdlib.h>
 #include <stdio.h>
+#include <string.h>
 #include <unistd.h>
 #include <fcntl.h>
 #include <vte/vte.h>
diff -ur grip-3.3.1/src/tray.c grip-3.3.1_warn/src/tray.c
--- grip-3.3.1/src/tray.c	2005-01-29 19:34:17.000000000 +0100
+++ grip-3.3.1_warn/src/tray.c	2010-10-17 21:59:17.000000000 +0200
@@ -21,6 +21,7 @@
  */
 
 #include "tray.h"
+#include "common.h"
 #include "../pixmaps/rip1.xpm"
 #include "../pixmaps/menuplay.xpm"
 #include "../pixmaps/menupause.xpm"
